### PR TITLE
Ddp 7967 cmi pancan social media link updates

### PR DESCRIPTION
**Pull request by nsakande**
https://broadinstitute.atlassian.net/jira/software/c/projects/DDP/boards/836?modal=detail&selectedIssue=DDP-7967

### CODE REVIEW

File: pepper-apis/studybuilder-cli/studies/brain/emails/about_you_completed.html [hunk 1]
```diff
@@ -118,10 +118,12 @@
...
       <p style="font-family: 'Roboto', sans-serif; text-align: left; font-weight: 300; font-size: 15px; line-height: 23px;">The
         Brain Tumor Project Team</p>
-      <p style="font-family: 'Roboto', sans-serif; text-align: left; font-weight: 300; font-size: 15px; line-height: 23px;">For
-        additional updates, follow us on Twitter <a href="https://twitter.com/BrainTumorProj" target="_blank" style="color: #7154ff; text-decoration: none; cursor: pointer;">@BrainTumorProj</a>
+      <p style="font-family: 'Roboto', sans-serif; text-align: left; font-weight: 300; font-size: 15px; line-height: 23px;">
+        For additional updates, follow us on Twitter
+        <a href="https://twitter.com/BrainTumorProj" target="_blank" style="color: #7154ff; text-decoration: none; cursor: pointer;">@BrainTumorProj</a>
         or like our Facebook page, <a href="https://www.facebook.com/braintumorproject" target="_blank" style="color: #7154ff; text-decoration: none; cursor: pointer;">The
-          Brain Tumor Project</a>.</p>
+          Brain Tumor Project</a>.
```
Comment (leonidrakitin) [inline pepper-apis/studybuilder-cli/studies/brain/emails/about_you_completed.html:125]: How is this related to pancan?

File: pepper-apis/studybuilder-cli/studies/fon/activities/medication-log.conf [hunk 2]
```diff
@@ -88,11 +88,27 @@
             },
             "trueTemplate": {
               "templateType": "TEXT"
-              "templateText": "$options.yes"
```
Comment (leonidrakitin) [inline pepper-apis/studybuilder-cli/studies/fon/activities/medication-log.conf:91]: and also these changes who related to social media link updates?

File: pepper-apis/studybuilder-cli/studies/brain/emails/about_you_completed.html (hunk 1 above)
Comment (nsakande) [reply pepper-apis/studybuilder-cli/studies/brain/emails/about_you_completed.html:125]: links in pancan emails are changed
